lib/kafka/client.rb

```diff
@@ -49,23 +49,36 @@ class Client
     #
     # @param sasl_gssapi_keytab [String, nil] a KRB5 keytab filepath
     #
+    # @param sasl_scram_username [String, nil] SCRAM username
+    #
+    # @param sasl_scram_password [String, nil] SCRAM password
+    #
+    # @param sasl_scram_mechanism [String, nil] Scram mechanism ("sha256", "sha512")
+    #
+    # @param use_ssl [Booleanm false] Use SSL
```

Comment:
Why add this parameter? The decision on whether to use SSL is already done here:

https://github.com/zendesk/ruby-kafka/blob/a0e1d14c9a5c3f2c653f73bada5c08d9d957cc5f/lib/kafka/client.rb#L528-L529